enterprise/__init__.py

```diff
@@ -2,4 +2,4 @@
 Your project description goes here.
 """
 
-__version__ = "4.3.3"
+__version__ = "4.3.2"
```

Comment:
@irtazaakram I think you accidently reduced the version number instead of increasing it. Also, add and entry in [CHANGELOG](https://github.com/openedx/edx-enterprise/blob/master/CHANGELOG.rst) file.

Reply:
This one has already been used and is currently deployed on PyPI https://github.com/openedx/edx-enterprise/releases/tag/4.3.2

Reply:
🤦 Fixed it and updated changelog